crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -554,11 +554,17 @@ impl<'w, 's, T: GizmoConfigGroup> Gizmos<'w, 's, T> {
     /// # bevy_ecs::system::assert_is_system(system);
     /// ```
     #[inline]
-    pub fn rect_2d(&mut self, position: Vec2, rotation: f32, size: Vec2, color: Color) {
+    pub fn rect_2d(
+        &mut self,
+        position: Vec2,
+        rotation: impl Into<Rotation2d>,
```

Comment:
IMHO, using `impl Into<Rotation2d>` everywhere is a bit overkill.

Reply:
The only disadvantage is probably that it might make the function signature look slightly more complicated.

It does add a bit of context and explicitness though, since `Rotation2d` should always be counterclockwise and in radians. If we also added more methods that maybe return 2D rotations, you'd be able to pass them to methods directly instead of first converting to radians and then having the methods convert them back to `Rotation2d` anyway.

In my mind, `Rotation2d` is the 2D equivalent of a `Quat`, and should be used everywhere for rotation similarly. Taking an `impl Into<Rotation2d>` can be more efficient and flexible.

Reply:
Agree. Into<X> is a powerful abstraction that we should take full advantage of.